From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Herman van Hazendonk <github.com@herrie.org>
Date: Sat, 13 Sep 2026 15:10:00 +0200
Subject: [PATCH] sysbus: grant mediaindexer the settings query ACG

localeobserver.h:40 subscribes to
luna://com.webos.settingsservice/getSystemSettings to follow the locale,
which is protected by settings.query.

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
Upstream-Status: Pending
---
 files/sysbus/com.webos.service.mediaindexer.perm.json.in | 3 ++-
 1 file changed, 2 insertion(s), 1 deletion(s)

--- a/files/sysbus/com.webos.service.mediaindexer.perm.json.in
+++ b/files/sysbus/com.webos.service.mediaindexer.perm.json.in
@@ -2,7 +2,8 @@
     "com.webos.service.mediaindexer": [
         "mediadatabase.operation",
         "mediadatabase.management",
-        "physicaldevice.query"
+        "physicaldevice.query",
+        "settings.query"
     ],
     "com.webos.service.mediaindexer.media": [
         "mediadatabase.operation",
